README.md

```diff
@@ -37,9 +35,47 @@ multiple Terraform modules, remote state, and locking.
    Terraform you already have installed. However, based on the settings in your `terraform.tfvars` file, Terragrunt can
    configure remote state, locking, extra arguments, and lots more.
 
+1. Terragrunt is a direct implementation of the ideas expressed in 
+   [Terraform: Up & Running](http://www.terraformupandrunning.com). Additional background reading that will help
+   explain the motivation for Terragrunt includes the Gruntwork.io blog posts
+   [How to create reusable infrastructure with Terraform modules](https://blog.gruntwork.io/how-to-create-reusable-infrastructure-with-terraform-modules-25526d65f73d)
+   and [How to use Terraform as a team](https://blog.gruntwork.io/how-to-use-terraform-as-a-team-251bc1104973).
+
+## Table of Contents
+
+1. [Install Terragrunt](#install-terragrunt)
+1. [Use cases](#use-cases)
+   1. [Keep your Terraform code DRY](#keep-your-terraform-code-dry)
+   1. [Keep your remote state configuration DRY](#keep-your-remote-state-configuration-dry)
+   1. [Keep your CLI flags DRY](#keep-your-cli-flags-dry)
+   1. [Work with multiple Terraform modules](#work-with-multiple-terraform-modules)
```

Comment:
I went with my suggestion "Execute Terraform commands on multiple modules at once". If you like that, then the PR is good to go. If you want to change it, just comment and I'll update accordingly.